test/test_plugins.py

```diff
@@ -68,6 +68,24 @@ def test_importing_zipped_module(self):
             os.remove(zip_path)
             importlib.invalidate_caches()  # reset the import caches
 
+    def test_plugin_locations(self):
```

Comment:
if we're renaming the `Config` var too, then let's go all out
```suggestion
    def test_plugin_dirs(self):
```